k8s/clusterrole.go

```diff
@@ -0,0 +1,36 @@
+package k8s
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+func UpdateClusterRoleBindingFromYaml(k8sClient *kubernetes.Clientset, clusterRoleBindingYaml string) error {
+	clusterRoleBinding := rbacv1.ClusterRoleBinding{}
+	decoder := yamlutil.NewYAMLToJSONDecoder(bytes.NewReader([]byte(clusterRoleBindingYaml)))
+	err := decoder.Decode(&clusterRoleBinding)
+	if err != nil {
+		return err
+	}
+	for retries := 0; retries <= 5; retries++ {
+		_, err = k8sClient.RbacV1().ClusterRoleBindings().Create(&clusterRoleBinding)
+		if err != nil {
```

Comment:
I'm sure I've seen this complicated loop in some other place in this PR :) I'd suggest you simplify it